man/systemd.service.xml

```diff
@@ -898,6 +898,18 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>RestartQuick=</varname></term>
+
+        <listitem><para>Takes a boolean value that specifies whether the service should restart quick,
+        i.e. without going through a failed state. Defaults to <option>no</option>.</para>
```

Comment:
this should make clear what is and what isn't skipped. i.e. say whether ExecStop= and such are invoked, and what about OnSuccess= and OnFailure=